app/views/static_pages/dashboard_steps/community_rules.md

```diff
@@ -7,45 +7,58 @@ Our moderation team is committed to providing a friendly, safe and welcoming env
+-   **Stay online after asking a question.** Most of the time, answers are not simple and straight-forward. Therefore, you should be prepared to stick around and discuss it with those trying to help.
 
-* **Stay online after asking a question.** Most of the time, answers are not simple and straight-forward. Therefore, you should be prepared to stick around and discuss it with those trying to help.
+-   **Wait patiently for a response.** Even though our community is fairly active, not all questions will be answered right away. While you are waiting, make sure that you have provided a detailed question and continue trouble-shooting. If your post is getting buried and/or if you have been waiting for an hour, you can direct people to your question.
 
-* **Wait patiently for a response.** Even though our community is fairly active, not all questions will be answered right away. While you are waiting, make sure that you have provided a detailed question and continue trouble-shooting. If your post is getting buried and/or if you have been waiting for an hour, you can direct people to your question.
+-   **Do not ask to ask a question.** You have permission to ask questions, provided that it falls within our rules. We know you mean well, but it is somewhat inconsiderate to make people wait for your actual question. For more information, please read [Don't ask to ask, just ask](https://dontasktoask.com/).
 
-* **Do not ask to ask a question.** You have permission to ask questions, provided that it falls within our rules. We know you mean well, but it is somewhat inconsiderate to make people wait for your actual question. For more information, please read [Don't ask to ask, just ask](https://dontasktoask.com/).
+-   **Use external services for code sharing.** Posting small snippets of code is acceptable, but anything longer than a few lines is hard to read and parse. Do not upload files for people to download. Use [Pastebin](http://pastebin.com/) for long error messages. Use [CodePen](https://codepen.io/), [Replit](https://replit.com/), or [CodeSandbox](https://codesandbox.io/) for code examples.
 
-* **Use external services for code sharing.** Posting small snippets of code is acceptable, but anything longer than a few lines is hard to read and parse. Do not upload files for people to download. Use [Pastebin](http://pastebin.com/) for long error messages. Use [CodePen](https://codepen.io/), [Replit](https://replit.com/), or [CodeSandbox](https://codesandbox.io/) for code examples.
+-   **Post screenshots when necessary, not photos.** If you need to share an image to provide additional context when seeking help, such as to show what your terminal or an error says, always post a screenshot using the default method for your computer or with a third party screenshot tool. Taking a picture of your screen from your phone provides a lower quality image, making it difficult for anyone to actually help.
```

Comment:
More great insight here. However, I think we should combine the two "do nots" together, because it feels buried inside the other guideline and end up making three bullet points. Also, I want to make sure you know that this information is expanded on in the proposed https://www.theodinproject.com/how_to_ask, under the "Provide the Relevant Source" section.
```suggestion
-   **Use external services to share code.** Posting small snippets of code is acceptable, but anything longer than a few lines is hard to read and parse. Use [CodePen](https://codepen.io/), [Replit](https://replit.com/), or [CodeSandbox](https://codesandbox.io/) to share code.

-   **Use screenshots to share other relevant information.** When you need to share an image of your terminal window or an error message, use a screenshot tool on your computer. If you are not familiar with this tool, you should research it using your favorite search engine. You can also use [Pastebin](http://pastebin.com/) to share long error messages.

-   **Do not share files or a picture of your computer screen.** Files can contain malware, so do not share files for people to download. A picture of a computer screen provides a low-quality image that is difficult for others to read.
```

Reply:
For this, the changes I made align with what you have here with some tweaks.

- I think including the Pastebin link with the "external sources" bullet makes more sense as they're all related in sharing text snippets.

- I updated the screenshot text to what you have, but kept the "don't share photos" verbiage there as well (as opposed to the files bullet below it). Again, I think it's more related to screenshots than file uploads.

Let me know what you think of the updated verbiage and bullets in this section.

Reply:
Cool. I think the pastebin link fits in either place and debated it back and forth myself.